examples/igbh/dist_train_rgnn.py

```diff
@@ -204,8 +204,8 @@ def run_training_proc(local_proc_rank, num_nodes, node_rank, num_training_procs,
       loss.backward()
       optimizer.step()
       total_loss += loss.item()
-      train_acc += sklearn.metrics.accuracy_score(y.cpu().numpy(),
-          out.argmax(1).detach().cpu().numpy())*100
+      train_acc += sklearn.metrics.accuracy_score(y.clone().cpu().numpy(),
+          out.argmax(1).detach().clone().cpu().numpy())*100
```

Comment:
Unnecessary to clone the labels here? As `sklearn.metrics.accuracy_score()` only calculates the accuracy and will not cache/store the input parameters.

Reply:
done